MergerLogic/Clients/S3Client.cs

```diff
@@ -82,11 +83,17 @@ private bool IsKeyError(Exception e)
         {
             string methodName = MethodBase.GetCurrentMethod().Name;
             this._logger.LogDebug($"[{methodName}] start z: {z}, x: {x}, y: {y}");
-            string keyPrefix = this._pathUtils.GetTilePathWithoutExtension(this.path, z, x, y, true);
+            string keyPrefix = this._pathUtils.GetTilePath(this.path, z, x, y, TileFormat.Jpeg, true);
+
             byte[]? imageBytes = this.GetImageBytes(keyPrefix);
             if (imageBytes == null)
             {
-                return null;
+                keyPrefix = this._pathUtils.GetTilePath(this.path, z, x, y, TileFormat.Png, true);
```

Comment:
The common use case of get tile is mostly for PNG (Update), consider changing the order PNG first and Jpg after

Reply:
No, the common use-case is that the target is `jpeg`, this is why we attempt for this first.
Common meaning the most updated and not the most of layer type.